channels/ly.m3u

```diff
@@ -1,5 +1,21 @@
 #EXTM3U x-tvg-url="https://raw.githubusercontent.com/Fazzani/grab/master/merge.zip"
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://libyaschannel.com/wp-content/uploads/2015/01/uclogo.png" group-title="",Abnae Al balad
+https://master.starmena-cloud.com/hls/abna.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://www.ljbctv.tv/logo.png" group-title="",Jamahiria TV
+https://master.starmena-cloud.com/hls/jam.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/N9dO8AE.png" group-title="",Libya 218
 https://stream.218tv.net/libya218TV/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/mzwdt39.jpg" group-title="News",Libya 218 News
 http://95.85.47.43/libya218news/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/mzwdt39.jpg" group-title="News",Libya 218 News
+https://stream.218tv.net/libya218news/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://pbs.twimg.com/profile_images/953959542928412673/wYWx5MRY.jpg" group-title="",Libya Al Ahrar TV
+https://video.zidivo.com/live983/GrtjM_FNGC/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="http://www.elahmad.com/tv/images/libyaalhadath.jpg" group-title="",Libya Al Hadath
+https://master.starmena-cloud.com/hls/hd.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://www.banat5.com/wp-content/uploads/2019/10/channel-8.jpg" group-title="",Libya Almostaqbal
+http://livedroos.com:8080/live/Libyah-Mostaqbal/tracks-v1a1/mono.m3u8
```

Comment:
Not working.

Reply:
Maybe a non 24-7 stream. Maybe @link8k should try on its own and come back with proof (image or stream data)

Reply:
Apologies, I remember testing them (maybe it went down). I will add https://streamtest.in/ links next time